문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    var answer = false;
+    let counts = {
+        p: 0,
+        y: 0,
+    };
+    for (var i = 0; i <= s.length; i++) {
+        s[i] === "P" || s[i] === "p" ? (counts["p"] += 1) : counts;
+        s[i] === "Y" || s[i] === "y" ? (counts["y"] += 1) : counts;
+    }
+    counts["p"] === counts["y"] ? (answer = true) : (answer = false);
```

Comment:
여기서 변수를 선언하고 삼항연산자는 안써도 될거 같아요
```suggestion
    const answer = counts["p"] === counts["y"];
```